### ISSUE TITLE
"Cannot cast mixed to float." static analysis error for "traces_sample_rate" config key

### ISSUE THREAD

**Issue by khalyomede** [getsentry/sentry-php#1438]
### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

^3.3

### Steps to Reproduce

1. Run phpstan
2. An error occurs

> Cannot cast mixed to float.

### Expected Result

No static analysis errors.

### Actual Result

> Cannot cast mixed to float.

**Comment by stayallive** [comment]
A little more context of the error (like the code it's erroring on) is appreciated!

phpstan shouldn't analyze vendor code so this error occurs in your own code most likely.

**Comment by khalyomede** [comment]
Sorry, I find the issue template a bit restricting imo, was not sure where to put additional context.

I ran phpstan on my config file, and after opening the issue I was able to fix it simply by calling floatval() (which allows to take a mixed type) instead of float cast language expression.

I'm able to open a pull request if you want. If you consider this is not needed since config file should not be statically analyse, feel free to close it.
